addons/binding/org.openhab.binding.zway/ESH-INF/i18n/zway_de.properties

```diff
@@ -1,20 +1,20 @@
 # binding
 binding.zway.name = Z-Way Binding
-binding.zway.description = Das Z-Way-System ist ein Softwarepaket für den Z-Wave Funkstandard. Das System besteht im wesentlichen aus einer Firmware für Z-Wave Transceiver, einer Kommunikations- und einer Automatisierungskomponente zur Steuerung und Konfiguration des Netzwerks. Das Z-Way Binding nutzt die HTTP-Schnittstelle um alle Geräte zu laden und im Discovery-Prozess zur Verfügung zu stellen.<br>Neben einem Polling der Gerätezustände werden optional die resultierenden Geräte als Observer im Z-Way System registriert, um Zustandsänderungen der Geräte zu erhalten (dazu wird aktuell die Z-Way App <i>OpenHAB Konnektor</i> benötigt).
+binding.zway.description = Das Z-Way-System ist ein Softwarepaket f�r den Z-Wave Funkstandard. Das System besteht im wesentlichen aus einer Firmware f�r Z-Wave Transceiver, einer Kommunikations- und einer Automatisierungskomponente zur Steuerung und Konfiguration des Netzwerks. Das Z-Way Binding nutzt die HTTP-Schnittstelle um alle Geräte zu laden und im Discovery-Prozess zur Verf�gung zu stellen.<br>Neben einem Polling der Ger�tezust�nde werden optional die resultierenden Ger�te als Observer im Z-Way System registriert, um Zustands�nderungen der Ger�te zu erhalten (dazu wird aktuell die Z-Way App <i>OpenHAB Konnektor</i> ben�tigt).

```

Comment:
Sorry. I've probably looked at it too fleetingly. I have now read everything again exactly.